packages/awslint/.eslintrc.js

```diff
@@ -1,3 +1,168 @@
-const baseConfig = require('cdk-build-tools/config/eslintrc');
-baseConfig.parserOptions.project = __dirname + '/tsconfig.json';
-module.exports = baseConfig;
+// This cannot reference the build rules from cdk-build-tools as this
```

Comment:
The content here is not equivalent to the imported `eslintrc.js` anymore (the style part has evolved). I understand the circular dependency issue here, how about moving the `/config` part of `cdk-build-tools` to a separate package `cdk-build-tools-config`?